vaadin-grid-theme.html

```diff
@@ -0,0 +1,76 @@
+<link rel="import" href="vaadin-grid-active-item-behavior.html">
+<link rel="import" href="vaadin-grid-selection-behavior.html">
+<link rel="import" href="vaadin-grid-data-source-behavior.html">
+<link rel="import" href="vaadin-grid-row-details-behavior.html">
+
+<dom-module id="vaadin-grid-theme-styles">
+  <template>
+    <style>
+
+    /* Default borders */
+
+    thead tr:last-child th ::content > .cell-content {
+      border-bottom: 1px solid var(--divider-color, rgba(0, 0, 0, 0.08));
```

Comment:
Using a semi-transparent color together with CSS transforms (3D in particular I suppose) causes the line to be blurred on sub-pixels in Safari.

Not a blocker by any means, but I felt like I should report that issue here. I was about to make a new explicit issue for it, but I think it’s not that critical actually, as most users will modify the theme in any case.